azext_iot/central/providers/monitor_provider.py

```diff
@@ -90,15 +91,15 @@ def _build_targets(
         return targets
 
     def _build_handler(
```

Comment:
Would recommend using python black for auto formatting.

Reply:
changed to black and re-run formatting on the folder